### DISCUSSION TITLE
PayPerShare / PPS

### DISCUSSION THREAD

**Discussion by Roddy1219** [coinfoundry/miningcore#863] [Ideas]
## PPS Support
Hope to support PPS mode

**Comment by bitspill** [comment]
Create and write a PPS payment handler if you know C#, otherwise only Solo and PPLNS are supported currently


Edit: originally ready that as "how to support pps" rather than hope, but the answer is largely the same. It's generally frowned upon scheme and thus far no one has written it

**Comment by oliverw** [comment]
Like @bitspill said, feel free to submit a PR. Until then closing.
